modules/nw-enabling-a-provisioning-network-after-installation.adoc

```diff
@@ -0,0 +1,83 @@
+----
+$ oc get provisioning -o yaml > enable-provisioning-nw.yaml
+----
+
+. Modify the provisioning CR file.
++
+[source,terminal]
+----
+$ vim ~/enable-provisioning-nw.yaml
+----
++
+Scroll down to the `provisioningNetwork` configuration setting and change it from `Disabled` to `Managed`. Then, add the `provisioningOSDownloadURL`, `provisioningIP`, `provisioningNetworkCIDR`, `provisioningDHCPRange`, `provisioningInterface`, and `watchAllNameSpaces` configuration settings after the `provisioningNetwork` setting. Provide appropriate values for each setting.
++
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: metal3.io/v1alpha1
+  kind: Provisioning
+  metadata:
+    name: provisioning-configuration
+  spec:
+    provisioningNetwork: <1>
+    provisioningOSDownloadURL: <2>
+    provisioningIP: <3>
+    provisioningNetworkCIDR: <4>
+    provisioningDHCPRange: <5>
+    provisioningInterface: <6>
+    watchAllNameSpaces: <7>
+----
++
+Where:
++
+<1> The `provisioningNetwork` is one of `Managed`, `Unmanaged`, or `Disabled`. When set to `Managed`, Metal3 manages the provisioning network and the CBO will deploy the Metal3 pod with a configured DHCP server. When set to `Unmanaged`, the system administrator is responsible for configuring DHCP.
++
+<2> The `provisioningOSDownloadURL` is a valid HTTPS URL with a valid sha256 checksum that enables the Metal3 pod to download a qcow2 operating system image ending in `.qcow2.gz` or `.qcow2.xz`. This field is required whether the provisioning network is `Managed`, `Unmanaged`, or `Disabled`. For example: `http://192.168.0.1/images/rhcos-48.83.202103221318-0-openstack.x86_64.qcow2.gz?sha256=323e7ba4ba3448e340946543c963823136e1367ed0b229d2a05e1cf537642bb8`.
++
+<3> The `provisioningIP` is the static IP address that the DHCP server and ironic will be using to provision the network. This static IP address should be within the `provisioning` subnet, and outside of the DHCP range. If set, it must have a valid IP address even if the `provisioning` network is `Disabled`. The static IP address is bound to the metal3 pod. If the metal3 pod dies and moves to another server, the static IP address moves to the new server too.
++
+<4> The Classless Inter-Domain Routing (CIDR) address. If set, it must have a valid CIDR address even if the `provisioning` network is `Disabled`. For example: `192.168.0.1/24`.
++
+<5> The DHCP range. This setting is only applicable to a `Managed` provisioning network. Omit this configuration setting if the `provisioning` network is `Disabled`. For example: `192.168.0.64, 192.168.0.253`.
++
+<6> The NIC name for the `provisioning` interface on cluster nodes. This setting is only applicable to `Managed` and `Unamanged` provisioning networks. Omit this configuration setting if the `provisioning` network is `Disabled`.
++
+<7> The default value is `false`. Set this setting to `true` if you want metal3 to watch namespaces other than the default `openshift-machine-api` namespace.
```

Comment:
Suggest flipping the second sentence around to get the goal first, i.e. "To watch namespaces [...]"